docs/changes/units/17232.bugfix.rst

```diff
@@ -0,0 +1,6 @@
+Parsing custom units with ``u.Unit()`` using the ``"vounit"`` format now obeys
+the ``parse_strict`` parameter, unless the custom units are made explicit with
+quotation marks.
+For example, ``u.Unit("custom_unit", format="vounit")`` now raises an error,
```

Comment:
I would guess that explicitly calling `u.Unit(..., format="vounit") is not very common and that the code in question is much more frequently used indirectly when reading in VOTables with `astropy`. But the `astropy` VOTable reader explicitly sets `parse_strict="silent"` instead of relying on the default, so the change in the default behavior might have a very small impact in practice.

Reply:
OK, so today I double-checked and previously we got a warning, while we get an error now. So based on that, this should have never been a backport candidate.

As for usage, we never really know, right? E.g I see usage in asdf, and in the lsst repos, too.